recipe/meta.yaml

```diff
@@ -27,31 +27,35 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake
     - make  # [unix]
+    - ninja
   host:
     - python
-    - setuptools
-    - blosc
```

Comment:
Rather then building with the vendored `blosc`, are we able to build against our package using [USE_SYSTEM_BLOSC=1](https://github.com/Blosc/python-blosc#compiling-with-an-installed-blosc-library)?

Reply:
I'll try to rebuild with that flag

Reply:
I've rebuilt it using `USE_SYSTEM_BLOSC=1` and it succeeded